src/components/WebexMemberRoster/WebexMemberRoster.jsx

```diff
@@ -23,12 +23,47 @@ import useMemberships from '../hooks/useMemberships';
  * @returns {object} JSX of the component
  */
 export default function WebexMemberRoster({destinationID, destinationType}) {
-  const memberships = useMemberships(destinationID, destinationType);
-  const members = memberships.map(
+  const members = useMembers(destinationID, destinationType);
+  const inMeetingFilter = (member) => member.inMeeting;
+  const notInMeetingFilter = (member) => !member.inMeeting;
+
+  const renderMembers = (members) => members.map(
     (member) => <WebexMember key={member.personID} personID={member.personID} />,
   );
 
-  return <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-roster`}>{members}</div>;
+  const getMembers = () => {
+    if (destinationType !== 'meeting') {
```

Comment:
Please use [the enum](https://github.com/webex/component-adapter-interfaces/blob/master/src/MembershipsAdapter.js#L31) (by importing it) instead of hardcoding strings